Consul.Test/OperatorTest.cs

```diff
@@ -82,5 +82,12 @@ public async Task Operator_KeyringInstallListPutRemove()
                 Assert.True(response.Keys.ContainsKey(newKey));
             }
         }
+
+        [SkippableFact]
+        public async Task Operator_GetLicense()
+        {
+            var license = await _client.Operator.GetConsulLicense();
+            Skip.If(license.StatusCode == System.Net.HttpStatusCode.NotFound, "This test requires an Enterprise Consul version");
```

Comment:
This is not the most elegant, but it does the job. What do you guys think?

Consul Enterprise APIs, when called on the community edition server naturally return HTTP 404. 

With a XUnit `SkippableFact`, we can implement tests for these Enterprise version-only APIs and they can be skipped by adding this 2 line check in the beginning of any of these tests when the server is not Enterprise.

Reply:
Another possibility is to implement custom `Fact` attribute, e.g.:
```
    public class EnterpriseOnlyFactAttribute : FactAttribute
    {
        public EnterpriseOnlyFactAttribute()
        {
            if (string.IsNullOrEmpty(Environment.GetEnvironmentVariable("RUN_CONSUL_ENTERPRISE_TESTS")))
            {
                Skip = "Ignored non-0enterprise servers";
            }
        }
    }
```
And the use:
```
[EnterpriseOnlyFactAttribute]
public async Task Operator_GetLicense()
{
    var license = await _client.Operator.GetConsulLicense();
    ...
}

It lets us to skip enterprise-only facts without making any extra http requests. 
Also, it feels more elegant in my opinion.

Reply:
Much better, thanks!